prediction_market_agent_tooling/markets/metaculus/metaculus.py

```
@@ -66,14 +67,15 @@ def from_data_model(model: MetaculusQuestion) -> "MetaculusAgentMarket":
 
     @staticmethod
-    def get_markets(
+    def get_markets(  # type: ignore[override]
         limit: int,
         sort_by: SortBy = SortBy.NONE,
         filter_by: FilterBy = FilterBy.OPEN,
         created_after: t.Optional[DatetimeUTC] = None,
         excluded_questions: set[str] | None = None,
         tournament_id: int | None = None,
-        fetch_scalar_markets: bool = False,
+        market_types: list[MarketType] = [MarketType.ALL],
```

Ruff (0.11.9)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)